SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -256,7 +256,7 @@ void USpatialActorChannel::RetireEntityIfAuthoritative()
 			}
 			else
 			{
-				Sender->RetireEntity(EntityId, Actor->IsNetStartupActor());
+				NetDriver->ActorSystem->RetireEntity(EntityId, Actor->IsNetStartupActor());
```

Comment:
Lots of removal of sender from actor channel, can we remove the member variable now also?

Reply:
not yet, need to deal with RPCs etc in upcoming PRs